src/benchq/problem_embeddings/_qsp.py

```diff
@@ -189,10 +301,8 @@ def _replace_named_qubits(
 
 
 def _replace_gate(op: cirq.Operation) -> Optional[cirq.Operation]:
-    if isinstance(op.gate, cirq.YPowGate) and op.gate.exponent == 0.5:
+    if isinstance(op.gate, cirq.YPowGate) and abs(op.gate.exponent) == 0.5:
```

Comment:
Is this correct? Don't we have both these cases because there's a minus sign on the case you deleted?